epub34/a11y-tech/index.html

```diff
@@ -505,11 +498,10 @@ <h4>Do not repeat roles across chunked content</h4>
 						is rare, at least for books, for an EPUB publication to contain only one EPUB content document
 						with all the content in it.</p>
 
-					<p>When content is chunked in this way, it often requires the <a
-							href="https://www.w3.org/TR/epub/#dfn-epub-creator">EPUB creator</a> to make decisions about
-						how best to restructure the information. A part, for example, will typically not include all the
-						chapters that belong to it. The EPUB creator will instead separate the part heading from each
-						chapter, putting each into a separate document.</p>
+					<p>When content is chunked in this way, it often requires tough decisions about how best to
+						restructure the information. A part, for example, will typically not include all the chapters
+						that belong to it. Instead, the part heading might be separated from each chapter, leaving each
+						chapter in a separate document.</p>
```

Comment:
```suggestion
					<p>When content is chunked in this way, it often requires tough decisions about how best to
						restructure the information. A part, for example, will typically not include all the chapters
						that belong to it. Instead, the part heading might be separated from each chapter, leaving each
						chapter in a separate document.</p>
```

I do not have a problem with this text, but I just remark that the word "tough" adds an emphasis to the sentence that the original does not really have. But I am o.k. keeping this if you think it is appropriate.

Reply:
Ya, but if it was an easy decision then there'd be nothing to document... 😄 

We could change the focus away from the decision making with: "it often requires restructuring the information to fit within the new file structure". That's more what the technique is about.

```suggestion
					<p>When content is chunked in this way, it often requires restructuring the information to fit
						within the new file structure. A part, for example, will typically not include all the chapters
						that belong to it. Instead, the part heading might be separated from each chapter, leaving each
						chapter in a separate document.</p>
```